### ISSUE TITLE
VAC BAN wave!

### ISSUE THREAD

**Issue by Is82jM** [Jire/Charlatano#184]
Over 5k vac bans after new update... I used Charlatano on 2 acc before this update and they are not banned. But is it still UD after the update? @Jire 
https://www.vac-ban.com/

**Comment by RN3D** [comment]
I would wait for the new Offsets and more there is a 85% chance that the Offsets changed, using old Offsets could cause a Ban.

**Comment by wuwujia** [comment]
eager to know 20170607 updates would affect how badly this program

**Comment by SquareHeadd** [comment]
Did anyone banned ?, because I used after the update and didn't get a ban

**Comment by RN3D** [comment]
I tested it, didnt got Banned ... for sure i would just use visuals no trigger and no aim

**Comment by ricabook** [comment]
I'm using ESP and AIM and didn't got banned.

**Comment by w-o-a-h** [comment]
"Using old Offsets could cause a Ban." @RN3D you KNOW nothing about what you are talking about.

**Comment by RN3D** [comment]
@w-o-a-h Yes, i do .. fetching old Offsets can be easy got dt. by some Anti Cheats ...

**Comment by w-o-a-h** [comment]
@RN3D all of the known anticheats don't do that all that can happen is a crash of the cheat if external or if they are internal the game crashes and if they use a driver they bsod/computer crashes.

**Comment by Yeezxc** [comment]
What about the unknown?
